search-api/requirements.txt

```diff
@@ -62,5 +62,7 @@ six==1.16.0
 strict-rfc3339==0.7
```

Comment:
you need to add changes here into requirements/... file. Otherwise next time someone builds the reqs to upgrade everything it will remove these

Reply:
I'd recommend upgrading to Poetry :P